src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -50,7 +51,8 @@ class AmendmentTableImpl : public AmendmentTable
     core::Clock::time_point m_lastReport;  // close time of most recent majority report
     beast::Journal m_journal;
 
-    AmendmentState* getCreate (uint256 const& amendment, bool create);
+    AmendmentState& getCreate (uint256 const& amendment);
+    AmendmentState* getExisting (uint256 const& amendment);
```

Comment:
Why does `getCreate` return a reference and `getExisting` a pointer? That seems almost reversed in a way, but I'd argue they should both return the same type.